modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -14579,6 +14579,10 @@ set-default-shipping-address=Set Default Shipping Address
 set-element-as-click-target=Set Element as Click Target
 set-expiration-date=Set Expiration Date
 set-filters=Set Filters
+set-lar-cleanup-job-interval=LAR Cleanup Job Execution Frequency
+set-lar-cleanup-job-interval-help=Configure execution time intervals for the LAR cleanp job in minutes. The default is every 15 minutes.
+set-lar-expiry-in-days=LAR Expiry
+set-lar-expiry-in-days-help=Successful export/import entries will be removed after the entered amount of days. If you want to keep the LAR files indefinitely, please set the value to -1 which is the default value.
```

Comment:
According to the requirements, we should also need a warning for the value `0`, which was mentioned in the story as: 
```
Have a help text to warn that "setting the time amount to 0 means the successful export/import entry will be deleted in the next scheduled cleanup, so there might be no chance to see what happened"
```